.eslintrc

```diff
@@ -9,6 +9,9 @@
     "mocha": true,
     "node": true
   },
+  "globals": {
+    "step": "readonly"
+  },
```

Comment:
We can add globals here, helps avoid ex.
```
/* eslint-disable no-unused-expressions */
expect(await text(content).exists()).to.be.true
```

Reply:
I agree. there are many of those with expect. would it be possible to do so?

Reply:
the problem is not with "expect", because the result of its invocation is used by the following function invocation; the problem is with ".true" or similar. There is a plugin we can use: https://www.npmjs.com/package/eslint-plugin-chai-friendly or maybe we can think about using something different than chai (maybe assert...?)